<?xml version="1.0"?>

<launch>

   <node pkg="octomap_server" type="octomap_server_node" name="octomap_server" output="screen">
    <param name="resolution" value="0.05" />
    <!-- fixed map frame (set to 'map' if SLAM or localization running!) -->
    <param name="frame_id" type="string" value="map" />
    <!-- maximum range to integrate (speedup!) -->
    <param name="sensor_model/max_range" value="5.0" />
    <!-- data source to integrate (PointCloud2) -->
    <!--remap from="cloud_in" to="/local_metric_map/intermediate_point_cloud" /-->
    <remap from="cloud_in" to="/local_metric_map/merged_point_cloud" />
  </node>
  
  <node pkg="fremen" type="froctomap" name="froctomap"  output="screen">
    <param name="resolution" value="0.05" />
    <param name="colorFactor" value="0.8" />
  </node>
 
  <node pkg="fremen" type="transform" name="transform" output="screen"></node>

  <node pkg="fremen" type="froctomap_ActionServer" name="Froctomap_ActionServer" output="screen"></node>
  
</launch>
